taxonomy/emsi_client.py

```diff
@@ -41,19 +41,26 @@ def __init__(self, scope):
                 access and its values are dependant on EMSI API specifications. Example values
                 are `emsi_open` and `postings:us`.
         """
-        self.client = None
         self.scope = scope
 
+        self.client = EdxRestApiClient(
+            self.API_BASE_URL,
+            append_slash=self.APPEND_SLASH,
+            oauth_access_token=self.access_token,
+        )
+
     @property
     def cache_key(self):
         """
         Return the cache key, caching is done based on the endpoint name and the scope property.
         """
         return get_cache_key(endpoint='EMSI', scope=self.scope)
 
-    def get_oauth_access_token(self, client_id, client_secret, grant_type='client_credentials'):
+    def fetch_oauth_access_token(self, client_id, client_secret, grant_type='client_credentials'):
         """
-        Return the access token if it is cached otherwise hit the endpoint and get a new access token.
+        Fetch a new access token from EMSI API.
+
```

Comment:
this extra line can be removed and turned into one statement

Reply:
Done